deploy/terraform-datacommons-website/modules/gke/configure_cluster.sh

```diff
@@ -12,8 +12,13 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+if [[ $LOCATION =~ ^[a-z]+-[a-z0-9]+$ ]]; then
```

Comment:
nit: add a quick comment on what the regex is looking for

Reply:
done. thanks!